shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/adapter/config/ProxyClusterContainerConfigurationCreator.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.test.integration.env.container.atomic.adapter.config;
+
+import lombok.RequiredArgsConstructor;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Proxy cluster container configuration creator.
+ */
+@RequiredArgsConstructor
+public final class ProxyClusterContainerConfigurationCreator {
+    
+    /**
+     * Create adaptor container config.
+     * @return adaptor container config
+     */
+    public static AdaptorContainerConfiguration create() {
+        return new AdaptorContainerConfiguration(getProxyDatasourceName(), getMountedResource());
+    }
+    
+    private static String getProxyDatasourceName() {
+        return "";
+    }
+    
+    private static Map<String, String> getMountedResource() {
+        Map<String, String> result = new HashMap<>(2, 1);
```

Comment:
The element of map is one, why the initial size is `2`?

Reply:
done